ui_src/src/components/getStartedItem/index.js

```diff
@@ -29,8 +34,38 @@ const GetStartedItem = (props) => {
                     <img className="get-started-bg-img" src={bgGetStarted} alt="bgGetStarted" />
                     <img className="get-started-bg-img-bottom" src={bgGetStartedBottom} alt="bgGetStartedBottom"></img>
                     <div className="get-started-top">
+                        <div className="video-container">
+                            <div>
+                                <img className="black-ball" src={blackBall} alt="black-ball"></img>
+                                <img className="orange-ball" src={orangeBall} alt="orange-ball"></img>
+                                <VideoPlayer
+                                    url={'https://www.youtube.com/watch?v=kp_l2qFokso'}
+                                    style={{
```

Comment:
Add a class name and move the style to the CSS file

Reply:
VideoPlayer can't get className and thus I sent the style in this way